src/modules/land_detector/MulticopterLandDetector.h

```diff
@@ -81,19 +81,19 @@ class MulticopterLandDetector : public LandDetector
 		param_t maxClimbRate;
 		param_t maxVelocity;
 		param_t maxRotation;
-		param_t maxThrottle;
+		param_t minThrottle;
 		param_t minManThrottle;
-		param_t acc_threshold_m_s2;
+		param_t ff_acc_threshold;
```

Comment:
I totally agree, it was just adapted to the naming of the other parameter but I'll also change that.